requirements.txt

```diff
@@ -10,3 +10,4 @@ gitpython
 pygithub <2
 ruamel.yaml
 conda-forge-pinning
+vsts-python-api
```

Comment:
The Python package is just called `vsts`

Reply:
True, but this is a requirements.txt, so it should have the Python name, no?

Reply:
I think this is used for conda, not pip, though.

Reply:
Yeah [it's used for CI]( https://github.com/conda-forge/conda-smithy/blob/06722d9553e73fd0610ceb443117d8d1bfda0805/.travis.yml#L26 ). Though can see this is a bit confusing. Opened issue ( https://github.com/conda-forge/conda-smithy/issues/920 ) about dropping it.